src/styles/StyleUtils.ts

```diff
@@ -492,11 +450,10 @@ function getButtonBackgroundColorStyle(buttonState = CONST.BUTTON_STATES.DEFAULT
 /**
  * Generate fill color of an icon based on its state.
  *
- * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
- * @param {Boolean} isMenuIcon - whether this icon is apart of a list
- * @returns {Object}
+ * @param buttonState - One of {'default', 'hovered', 'pressed'}
+ * @param isMenuIcon - whether this icon is apart of a list
  */
-function getIconFillColor(buttonState = CONST.BUTTON_STATES.DEFAULT, isMenuIcon = false) {
+function getIconFillColor(buttonState: string = CONST.BUTTON_STATES.DEFAULT, isMenuIcon = false): string {
```

Comment:
Why not use `ValueOf<typeof CONST.BUTTON_STATES> = CONST.BUTTON_STATES.DEFAULT,`?

Reply:
Done